doc/en/api/1.0.0/gwcseed.yaml

```diff
@@ -0,0 +1,100 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache Seeding and Truncating
+  description: A layer is a published resource (feature type or coverage).
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /seed.json:
+    get:
+      operationId: seedGet
+      summary: Query's currently running GWC task
+      description: Query's and returns a json array of the status for all currently running task. Requires .json extension in the request. The array contains a set of long in the following order:[tiles processed, total number of tiles to process, number of remaining tiles, Task ID, Task status].
+        The returned task status will be one of -1 = ABORTED, 0 = PENDING, 1 = RUNNING, 2 = DONE
+      produces:
+        - text/json
+      responses:
+        200:
+          description: OK
+          examples:
+            text/json: |
+              {"long-array-array":[
+                [364832,2761733810,998564,16,1],
+                [373472,2761733810,975448,17,1],
+                [363760,2761733810,1001509,18,1]
+              ]}
+  /seed/{layer}.json:
```

Comment:
Are the xml and json endpoints really different enough to need seperate documentation? Can't you use a single endpoint and have two entries under produces / consumes?

Reply:
I consolidated where I could - the /rest/seed.json is the only endpoint that queries running task for all layers and it only accepts a json extension.